<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>934</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>SIMBA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="filterGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Filtros</string>
    </property>
    <widget class="QComboBox" name="filterOrder">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Ingrese Orden...</string>
     </property>
     <item>
      <property name="text">
       <string>Ingrese Orden...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Primer Orden</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Segundo Orden</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="filterType">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ingrese Tipo...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Bajos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Altos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Todo</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="filterType2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ingrese Tipo...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Bajos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Altos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Todo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pasa Banda</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notch</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QComboBox" name="unitsDDown_T">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>u</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ω</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ξ</string>
     </property>
    </widget>
    <widget class="QComboBox" name="unitsDDown_W">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>u</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="unitsDDown_psy">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>u</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="inputGroupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>710</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Entrada</string>
    </property>
    <widget class="QComboBox" name="inputType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ingrese Señal...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Senoide</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pulso</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="unitsDDown_A">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>u</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="unitsDDown_f">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>n</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>u</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="plotGroupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>710</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Ploteo</string>
    </property>
    <widget class="QComboBox" name="plotType">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ingrese Tipo de Gráfico...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Salida</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polos/Ceros</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="simButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>740</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>225</red>
          <green>225</green>
          <blue>225</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>225</red>
          <green>225</green>
          <blue>225</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>225</red>
          <green>225</green>
          <blue>225</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simular</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1184</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
